crates/sled/src/lib.rs

```diff
@@ -3,20 +3,25 @@
 //! # Examples
 //!
 //! ```
-//! let t = sled::Db::start_default("my_db").unwrap();
+//! use sled::{ConfigBuilder, Db, IVec};
+//!
+//! let config = ConfigBuilder::new().temporary(true).build();
```

Comment:
even though this doctest creates garbage, I'd prefer to make the top-level example in the docs be something that won't delete itself upon Drop, since people will run it and get mad that it doesn't keep their data across process invocations even though it's kind of obvious

Reply:
(please keep the call to `Db::start_default`)